Frameworks/UIKit/UIPasteboard.mm

```diff
@@ -281,47 +281,13 @@ + (WSSInMemoryRandomAccessStream*)_populateStream:(WSSInMemoryRandomAccessStream
 + (WSSInMemoryRandomAccessStream*)_grabStreamFromUIImage:(UIImage*)image {
```

Comment:
CF is behaving strangely. now i saw earlier comment. close this one.

---
In reply to: [87309387](https://github.com/Microsoft/WinObjC/pull/1332#discussion_r87309387) [](ancestors = 87309387)